src/plugins/URLTimeSettingsSynchronizer/URLTimeSettingsSynchronizer.js

```diff
@@ -106,30 +109,32 @@ export default class URLTimeSettingsSynchronizer {
   }
 
   setTimeApiFromUrl(timeParameters) {
-    if (timeParameters.mode === 'fixed') {
-      if (this.openmct.time.timeSystem().key !== timeParameters.timeSystem) {
-        this.openmct.time.timeSystem(timeParameters.timeSystem, timeParameters.bounds);
-      } else if (!this.areStartAndEndEqual(this.openmct.time.bounds(), timeParameters.bounds)) {
-        this.openmct.time.bounds(timeParameters.bounds);
+    const timeSystem = this.openmct.time.getTimeSystem();
+
+    if (timeParameters.mode === FIXED_MODE_KEY) {
```

Comment:
We have no way of setting the clock anymore. Perhaps create a new issue for that?